apps/client/src/components/User/UserProfileCard.svelte

```diff
@@ -1,25 +1,28 @@
 <script>
   export let user;
-  export let image;
-  import "iconify-icon";
-  import { Button, Card } from "svelte-ui";
-  import UserProfileCardControls from "./UserProfileCardControls.svelte";
+  export let githubUser;
+  export let currentUser;
+  import 'iconify-icon';
+  import { Card } from 'svelte-ui';
+  import avatar from '$lib/images/avatar.png';
+  import UserProfileCardControls from './UserProfileCardControls.svelte';
+
   const colorMap = {
-    isBountyHunter: "pink",
-    isCampusExpert: "blue",
-    isDeveloperProgramMember: "orange",
-    isEmployee: "blue",
-    isGitHubStar: "gold",
-    isHireable: "green",
+    isBountyHunter: 'pink',
+    isCampusExpert: 'blue',
+    isDeveloperProgramMember: 'orange',
```

Comment:
I think we have a problem with consistent formatting. We might have to fix the pre-commit hooks in a separate PR.